source/includes/fact-write-blocking-enable.rst

```diff
@@ -1,7 +1,9 @@
 When write-blocking is enabled, ``mongosync`` blocks writes:
 
-- On the destination cluster during sync
-- On the source cluster while committing
+- On the destination cluster during sync.
```

Comment:
I would also move the sentence `Write-blocking is not enabled by default.` to the top of this include and remove it from[ line 226 ](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/182/files#diff-a61a44aa44c4308e9fcfa894effe294e608468bf2a1e3cf752da6b039ef5f2beR226). This is a pivotal piece of information that the user must understand, even when not using reverse sync.

Reply:
Done!